### ISSUE TITLE
Images appear darkened

### ISSUE THREAD

**Issue by JustAlittleWolf** [Nettakrim/Signed-Paintings#17]
Images are displayed a lot darker in comparison to the image source

Minecraft image (with fullbright enabled):
![image](https://github.com/Nettakrim/Signed-Paintings/assets/54244277/9a16e349-e9e5-4177-abc8-129defdccecd)

Source image:
![image](https://github.com/Nettakrim/Signed-Paintings/assets/54244277/2039216f-df85-4fd1-99af-4074f2e0b2fd)

This happens for all images, not just specific ones.

This is a minor inconvenience, but it would be nice if it'd be fixed.

**Comment by JustAlittleWolf** [comment]
For me changing the normal Vector here:
https://github.com/Nettakrim/Signed-Paintings/blob/e4f5e4afa79d38be06c4a6e33417910bbc565aa3/src/main/java/com/nettakrim/signed_paintings/rendering/Cuboid.java#L81

to
```java
new Vector3f(0, 1, 0)
```
did the job, but that didn't work for my friend.

**Comment by Nettakrim** [comment]
interesting, i have a pretty long list of increasingly critical things to fix so ill be going through all the issues soon, makes sense there might be normal shenanigans

**Comment by Nettakrim** [comment]
![image](https://github.com/Nettakrim/Signed-Paintings/assets/106409257/8ab44b94-bb84-4cd1-8e76-e8f0942ee209)
ah the reason is because im using the same lighting that the signs themselves do - im not sure if this behaviour is desirable

**Comment by BondarenkoArtur** [comment]
I would say it's not a bug, it's a feature :grinning: 

It allows to make blinking / bright pictures as ads with simple redstone lamp

https://github.com/Nettakrim/Signed-Paintings/assets/9720950/93a15eb1-d1b9-40b8-9441-ec2fb1910fb8

**Comment by JustAlittleWolf** [comment]
@BondarenkoArtur well the lighting on signs is desired, but even at maximum light Level 15 they don't match the actual brightness of the same image when viewed in another program, in the browser for example.

**Comment by BondarenkoArtur** [comment]
Yes, I see, tried your suggestion, works for me, colors are better. But I'm personaly not clear how that works.
![image](https://github.com/Nettakrim/Signed-Paintings/assets/9720950/5fad48cc-36cb-42f3-ae7c-6052b860b1d6)
top image with same light level, bottom with high light level and offset.

**Comment by Nettakrim** [comment]
im pretty sure its because minecraft shades faces based on their angle, not because of block light level - block light does contribute, but even when in light level 15 an image may be darker based on its angle, which is what this issue is referring to

**Comment by BondarenkoArtur** [comment]
I was trying to put sign from all 4 sides of glowstone block, result seems dull for each of that :thinking:

**Comment by Nettakrim** [comment]
this is working as intended. insofar as it is currently consistent with minecraft

im pretty sure its nothing to do with light level, its because minecraft shades north/south faces lighter than east/west ones, and top faces are brighter still

you can easily see this with the new rotation sliders, or by just looking at some terrain
![java_CgaVXmzRBv](https://github.com/Nettakrim/Signed-Paintings/assets/106409257/47852546-76a8-464f-a88d-e532087f3c35)

**Comment by Nettakrim** [comment]
next version will have glow ink to make images fullbright (doing the normal trick) https://github.com/Nettakrim/Signed-Paintings/issues/52 - between that and placing light sources on images (which doesnt adjust the normals), that should cover all the cases
